### PR TITLE
[zh]Update content/zh/docs/reference/using-api/_index.md

### PR DESCRIPTION
**Pull request by JarHMJ**

### CODE REVIEW

File: content/zh/docs/reference/using-api/_index.md [hunk 1]
```diff
@@ -200,6 +201,7 @@ part is omitted, it is treated as if `=true` is specified. For example:
 例如：
  - 禁用 `batch/v1`， 对应参数设置 `--runtime-config=batch/v1=false`
  - 启用 `batch/v2alpha1`， 对应参数设置 `--runtime-config=batch/v2alpha1`
+ - 启用特殊版本的 API，如 `storage.k8s.io/v1beta1/csistoragecapacities`，对应参数设置`--runtime-config=storage.k8s.io/v1beta1/csistoragecapacities`
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/_index.md:204]:
```suggestion
 - 要启用特定版本的 API，如 `storage.k8s.io/v1beta1/csistoragecapacities`，可以设置
   `--runtime-config=storage.k8s.io/v1beta1/csistoragecapacities`
```

**Comment by tengqm** [comment]
Rather than having contributors randomly pick a page to sync, please check the issues [#33474](https://github.com/kubernetes/website/issues/33474), [#33475](https://github.com/kubernetes/website/issues/33475), [#33476](https://github.com/kubernetes/website/issues/33476), [#33477](https://github.com/kubernetes/website/issues/33477), [#33478](https://github.com/kubernetes/website/issues/33478), [#33479](https://github.com/kubernetes/website/issues/33479) and sign on tasks there. It would help save your time and avoid conflicts when working on different pages.

PRs like  this one are too small. They are wasting the CI gate resources and they are wasting time for reviewers.

File: content/zh/docs/reference/using-api/_index.md (hunk 1 above)
Comment (JarHMJ) [reply content/zh/docs/reference/using-api/_index.md]: done

File: content/zh/docs/reference/using-api/_index.md [hunk 2]
```diff
@@ -200,6 +201,8 @@ part is omitted, it is treated as if `=true` is specified. For example:
 例如：
  - 禁用 `batch/v1`， 对应参数设置 `--runtime-config=batch/v1=false`
  - 启用 `batch/v2alpha1`， 对应参数设置 `--runtime-config=batch/v2alpha1`
+ - 要启用特定版本的 API，如 `storage.k8s.io/v1beta1/csistoragecapacities`，可以设置
+ `--runtime-config=storage.k8s.io/v1beta1/csistoragecapacities`
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/_index.md:205]:
```suggestion
  `--runtime-config=storage.k8s.io/v1beta1/csistoragecapacities`
```

**Comment by JarHMJ** [comment]
> Rather than having contributors randomly pick a page to sync, please check the issues [#33474](https://github.com/kubernetes/website/issues/33474), [#33475](https://github.com/kubernetes/website/issues/33475), [#33476](https://github.com/kubernetes/website/issues/33476), [#33477](https://github.com/kubernetes/website/issues/33477), [#33478](https://github.com/kubernetes/website/issues/33478), [#33479](https://github.com/kubernetes/website/issues/33479) and sign on tasks there. It would help save your time and avoid conflicts when working on different pages.
> 
> PRs like this one are too small. They are wasting the CI gate resources and they are wasting time for reviewers.

ok, I see. I have signed some tasks on #33476.  Please merge this pr firstly。Thanks @tengqm ！

**Comment by JarHMJ** [comment]
Please review @tengqm. Thanks.

**Comment by tengqm** [comment]
205 行缩进不够，尽管属于吹毛求疵的问题，还是希望大家养成好习惯，做足功夫

**Comment by JarHMJ** [comment]
> 205 行缩进不够，尽管属于吹毛求疵的问题，还是希望大家养成好习惯，做足功夫

谢谢老师指导

**Comment by nameYULI** [comment]
related issue: https://github.com/kubernetes/website/issues/33478
reference-9
```
1	0	content/en/docs/reference/using-api/_index.md
```

**Pull request merged**